src/Atmos/Model/moisture.jl

```diff
@@ -83,23 +85,21 @@ function gradvariables!(m::EquilMoist, transform::Vars, state::Vars, aux::Vars,
   transform.moisture.h_tot = e_tot + R_m*T
 end
 
-
-function diffusive!(m::EquilMoist, diffusive::Vars, ∇transform::Grad, state::Vars, aux::Vars, t::Real, ρν::Union{Real,AbstractMatrix})
+function diffusive!(m::EquilMoist, diffusive::Vars, ∇transform::Grad, state::Vars, aux::Vars, t::Real, ρν::Union{Real,AbstractMatrix}, inv_Pr_turb::Real)
```

Comment:
Can we replace `∇transform` with `∇grad` and add an additional operator called `∇div` as well?

Reply:
I think that the these spatial operators need to be handled in the back end in order to hide DG details from the user. From my understanding, the purpose of this function is to compute `∇transform.f = grad(f)`. That way, we can add `flux.var += ∇transform.f` in `flux!`  for terms that appear in the form `div(grad(f))`.

Reply:
Maybe there is a case for splitting up the name convention this way, but it's a bit unclear to me.

Reply:
Note that `∇transform` isn't an operator, it just a variable name that happens to contain the spatial derivative of the variable `transform` from the `gradtransform` function.